perception/tensorrt_yolo/src/nodelet.cpp

```diff
@@ -134,6 +134,10 @@ void TensorrtYoloNodelet::callback(const sensor_msgs::msg::Image::ConstSharedPtr
 
   tier4_perception_msgs::msg::DetectedObjectsWithFeature out_objects;
 
+  if (!yolo::set_cuda_device(gpu_device_id_)) {
+    RCLCPP_ERROR(this->get_logger(), "Given GPU not exist or suitable");
```

Comment:
Is it fine not to return here even though gpu device id is invalid?

Reply:
Thanks, @wep21. Sorry, it's my fault. Without return, if the given GPU is invalid, tensorrt_yolo will work over GPU 0.